lib/file-store.js

```diff
@@ -33,10 +33,41 @@ FileStore.prototype.load = function (fn) {
     if (!json || json === '') {
       return fn(null, {})
     }
+
+    let store
     try {
-      fn(null, JSON.parse(json))
+      store = JSON.parse(json)
     } catch (err) {
-      fn(err)
+      return fn(err)
     }
+
+    // Check if old format and convert if needed
+    if (!store.hasOwnProperty('lastRun') && store.hasOwnProperty('pos')) {
+      if (store.pos === 0) {
```

Comment:
Unless I am mistaken, in pre-v1 position 0 was as if the first migration had run.  It was a zero-index into the array of migrations sorted by filename.  So it should be set to the first migration title here.

Reply:
I may not be understanding the <1.0 code correctly, but it looks like all migrations with an index less than `pos` have been run. The no migrations case has a `pos` of 0.

https://github.com/tj/node-migrate/blob/v0.2.4/lib/set.js#L33
https://github.com/tj/node-migrate/blob/v0.2.4/lib/set.js#L184

Reply:
Yep, looks like you are correct.  It had been a few months since I was in the pre-v1 code.  Sorry to send you down a false path.